frontend/src/scenes/batch_exports/batchExportLogic.ts

```diff
@@ -241,7 +243,17 @@ export const batchExportLogic = kea<batchExportLogicType>([
 
         batchExportRuns: [
             (s) => [s.batchExportRunsResponse],
-            (batchExportRunsResponse): BatchExportRun[] => batchExportRunsResponse?.results ?? [],
+            (batchExportRunsResponse): BatchExportRun[] => {
+                const runs = batchExportRunsResponse?.results ?? []
+                const deserializedRuns: BatchExportRun[] = runs.map((run) => ({
+                    ...run,
+                    data_interval_start: dayjs(run.data_interval_start),
+                    data_interval_end: dayjs(run.data_interval_end),
+                    created_at: dayjs(run.created_at),
+                    last_updated_at: run.last_updated_at ? dayjs(run.last_updated_at) : undefined,
+                }))
+                return deserializedRuns
```

Comment:
Not sure if this is the most "typescriptonic" way of doing this, but it didn't let me just `return runs.map((run) => convert(run))`